src/Pages/Wishlist/index.js

```diff
@@ -1,23 +1,117 @@
+import { useEffect } from 'react';
+import axios from 'axios';
+
+import { useNavigate } from 'react-router-dom';
+
 import './wishlist.css';
 import '../../styles/base.css';
 
 import { ProductCard } from '../../components/Card';
 
+import { useProduct } from '../../context/product-context';
+
+import { wrapAsync } from '../../utils/wrapAsync';
+
+import { addToCart } from '../../utils/productFeat';
+
 const Wishlist = () => {
+  const { productState, productDispatch } = useProduct();
+  let navigate = useNavigate();
+
+  useEffect(() => {
+    async function showWishList() {
+      const encodedToken = JSON.parse(
+        window.localStorage.getItem('encodedToken')
+      );
+
+      const response = await axios({
```

Comment:
create a sperate folder  there use axios , & return backend response here . 

because as application grow it will be very easy to debug